README.md

```diff
@@ -32,9 +33,20 @@ cd sourcery-api/
 // install dependencies
 bundle
 
+// create db and migrate
+rake db:{create,migrate}
```

Comment:
Totally cool with that! Would you mind pulling down this branch and making that change?